service/server/router_linux.cpp

```
@@ -102,10 +107,10 @@ bool RouterLinux::routeDelete(const QString &ipWithSubnet, const QString &gw, co
         return false;
     }
 
-    if (ip == "0.0.0.0") {
-        qDebug().noquote() << "Warning, trying to remove default route, skipping: " << ip << gw;
-        return true;
-    }
+ //   if (ip == "0.0.0.0") {
```

What are cases to remove default router?

---

This condition is not about default route.  
We need to remove tun2socks routes after disconnecting. This route can be with "0.0.0.0" ip. 

Changed this condition to case with default system route (with mask /0).